third_party/CMakeLists.txt

```diff
@@ -52,6 +56,7 @@ ExternalProject_Add(
   CMAKE_GENERATOR Ninja
   CMAKE_ARGS
     -DCMAKE_BUILD_TYPE=${CMAKE_BUILD_TYPE}
+    -DCMAKE_INSTALL_PREFIX=${METAL_INSTALL_PREFIX}
```

Comment:
This is needed because metal tries to install libfmt to /usr/bin which cannot be done without sudo.